hub/instruction_handling/create/generalities/all.tex

```diff
@@ -234,24 +234,41 @@
 		\]
+\end{description}
+Let us define the following shorthands:
+\[
+	\left\{ \begin{array}{lcl}
+		\locCreateUpfrontGasCost & \define & G_\text{create} + \locMxpGas              \\
+		\locCreateFullGasCost    & \define & \locCreateUpfrontGasCost + \locStpGasPoop \\
+	\end{array} \right.
+\]
+\begin{description}
 	\item[\underline{Setting the \gasCost{}:}]
 		we impose
 		\begin{enumerate}
 			\item \If $\locStaticx + \locMxpx = 1$              \Then $\gasCost _{i} = 0$
-			\item \If $\locOogx + \scenCreateUnexceptional = 1$ \Then $\gasCost _{i} = G_\text{create} + \locMxpGas$
+			\item \If $\locOogx + \scenCreateUnexceptional = 1$ \Then $\gasCost _{i} = \locCreateUpfrontGasCost$
 		\end{enumerate}
 	\item[\underline{Setting the \gasNext{}:}]
 		we impose
-		\begin{enumerate}
-			\item \If $\scenCreateException                 _{i} = 1$ \Then $\gasNext_{i} = 0$ (\trash)
-			\item \If $\scenCreateNoContextChange           _{i} = 1$ \Then $\gasNext_{i} = \gasActual_{i} - \gasCost_{i}$
-			\item \If $\scenCreateExecutionNonEmptyInitCode _{i} = 1$ \Then
-				\[
-					\gasNext_{i} = \gasActual_{i} - 
-					\left[ \begin{array}{cl}
-						+ & \gasCost_{i} \\
-						+ & \locStpGasPoop  \\
-					\end{array} \right]
-				\]
-		\end{enumerate}
+		\[
+			\hspace*{-0.5cm}
+			\left\{ \begin{array}{lcl}
+				\If \scenCreateException                 _{i} = 1 & \Then & \gasNext _{i} = 0 ~ (\trash)                               \\
+				\If \scenCreateAbort                     _{i} = 1 & \Then & \gasNext _{i} = \gasActual _{i} - \locCreateUpfrontGasCost \\
+				\If \scenCreateFCond                     _{i} = 1 & \Then & \gasNext _{i} = \gasActual _{i} - \locCreateFullGasCost    \\
+				\If \scenCreateExecutionEmptyInitCode    _{i} = 1 & \Then & \gasNext _{i} = \gasActual _{i} - \locCreateUpfrontGasCost \\
+				\If \scenCreateExecutionNonEmptyInitCode _{i} = 1 & \Then & \gasNext _{i} = \gasActual _{i} - \locCreateFullGasCost    \\
+			\end{array} \right.
+		\]
+		\saNote{}
+		We provide some details.
+		Exceptional \inst{CREATE}-type instructions, like any exceptional instruction, fully consume the currently available gas.
+		Aborted \inst{CREATE}-type instructions only pay for the \locCreateUpfrontGasCost{} (and there is no child context.)
+		\inst{CREATE}-type instructions that raise the \textbf{Failure Condition $\bm{F}$} pay the \locCreateFullGasCost{}, including the ``\texttt{(63/64)-ths}'' to the child context, and don't get any of it back.
+		\inst{CREATE}-type instructions that lead to execution pay for \locCreateFullGasCost{}.
+		However, if the provided initialization code is \emph{empty}, then the execution context doesn't change and gets the entire ``\texttt{(63/64)-ths}'' back immediately.
+		The effective gas cost is therefore the \locCreateUpfrontGasCost{}.
+		If the provided initialization code is \emph{nonempty} then the execution context pays the \locCreateFullGasCost{}.
+		It \emph{may} get some of it back later down the line.
```

Comment:
This is the fix. The issue was with **Failure Condition F** raising CREATE's not following the same pattern as other scenarios in `\scenCreateNoContextChange`

Reply:
For context, **Failure Condition F**: here we only care about the state collision bit, i.e. the first line

<img width="685" alt="image" src="https://github.com/user-attachments/assets/89dbbbc1-2f5f-422e-a758-69631afe8335">

Reply:
And the associated effect on returned gas
<img width="378" alt="image" src="https://github.com/user-attachments/assets/e6f44bcd-c69e-4578-b1e8-a0b82b118fba">